utils/near-stress-test/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use crate::cli::start_with_config;
+use clap::Clap;
+use near_chain_configs::GenesisValidationMode;
+use nearcore::get_default_home;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use tracing_subscriber::EnvFilter;
+
+mod cli;
+
+#[derive(Clap)]
+struct StressTestCmd {
+    #[clap(flatten)]
+    opts: NeardOpts,
+    #[clap(subcommand)]
+    subcmd: NeardSubCommand,
+}
+
+#[derive(Clap)]
+enum NeardSubCommand {
+    #[clap(name = "blocks")]
+    Blocks(BlocksCmd),
+    #[clap(name = "chunks")]
+    Chunks(ChunksCmd),
+}
+
+#[derive(Clap)]
+struct BlocksCmd {
+    requested_blocks_per_sec: usize,
+}
+
+#[derive(Clap)]
+struct ChunksCmd {
+    requested_chunks_per_sec: usize,
+}
+
+#[derive(Clap)]
+struct NeardOpts {
+    #[clap(long)]
+    home: Option<PathBuf>,
```

Comment:
@nagisa 
We use `NEAR_HOME` in case the argument is not specified.
Take a look at `get_default_home` inside

https://github.com/near/nearcore/pull/6385/files#diff-85e7bc5a479e2174607d12c9868c1e20d1aec31df63dff2949cab413a805eda2R77